modules/test/playwright/tests/frontend-editor-ckeditor-web/main/tests/ckeditor4/classic.spec.ts

```diff
@@ -160,17 +160,24 @@ test(
 			cardTitle: 'satellite.png',
 		});
 
-		await expect(ckeditor4Page.contextMenu.getByLabel('URL')).toHaveValue(
-			'/documents/d/guest/satellite-png'
+		const satelliteImageRegExp = new RegExp(
+			'/documents/d/guest/satellite-png.*'
```

Comment:
@markocikos didn't you change this very recently?

Reply:
Yes, that's what I said: some tests have been fixed already.